apps/cowswap-frontend/src/modules/volumeFee/state/cowswapFeeAtom.ts

```diff
@@ -16,7 +16,10 @@ const COWSWAP_VOLUME_FEES: Record<SupportedChainId, VolumeFee | null> = {
     bps: 10, // 0.1%
     recipient: '0x451100Ffc88884bde4ce87adC8bB6c7Df7fACccd', // Arb1 Protocol fee safe
   },
-  [SupportedChainId.BASE]: null,
+  [SupportedChainId.BASE]: {
+    bps: 10, // 0.1%
+    recipient: '0x3c4DBcCf8d80D3d92B0d82197aebf52574ED1F3B', // Base Protocol fee safe
```

Comment:
According to @blacktieaccountant.
See [this thread](https://cowservices.slack.com/archives/C05N1FNP3MX/p1741691454219959?thread_ts=1741689471.079029&cid=C05N1FNP3MX)

Reply:
no feature flag just in case?